docs/latest/deployment/recipes/embedded-viewer.md

```diff
@@ -49,6 +49,33 @@ window.config = {
 };
 ```
 
+```js
```

Comment:
Instead of duplicating config docs, we should link to `configuring` docs page.

Reply:
Updated.